bastion/task/cmd/backfill.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/project-nova/backend/bastion/config"
+	"github.com/project-nova/backend/pkg/abi/erc721"
+	"github.com/project-nova/backend/pkg/gateway"
+	"github.com/spf13/cobra"
+)
+
+var (
+	startId           int
+	endId             int
+	collectionAddress string
+)
+
+// backfillCmd represents the backfill command
+var backfillCmd = &cobra.Command{
+	Use:   "backfill",
+	Short: "backfill task for backfilling nft onchain data to database",
+	Long: `The task accepts flags: --start, ---end, --collection.
+--collection is required and will be used to determine which nft collection to backfill.
+--start and --end tell the task the range of nft token id to backfill. When they are specified. The default is to backfill all nft data from a colleciton.	
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if collectionAddress == "" {
+			fmt.Printf("[ERROR] collection address required. pass it with the flag --collection")
```

Comment:
do we have a common logging module?  Or you may create a github issue?  we used to use zerolog in previous project.

Reply:
Yes. I use zap from Uber. Didn't add logger here because I thought, for operation scripts, probably not much benefits to add it.  But I just added to make it consistent.